### PR TITLE
More cleanup

### PR DESCRIPTION
**Pull request by sjaeckel**
Please check the commit messages for details.

### CODE REVIEW

File: Makefile.am [hunk 1]
```diff
@@ -221,7 +221,7 @@ icons_sources = $(top_srcdir)/icons/*
 
 script_sources = bootstrap.sh
 
-man_sources = $(wildcard docs/profanity*.1)
+man1_sources = $(wildcard $(top_srcdir)/docs/*.1)
```
Comment (jubalh) [inline Makefile.am:224]: please mention in this commit message the commit that introduced the mistake. some explanation :)

File: .gitignore [hunk 2]
```diff
@@ -60,6 +64,9 @@ gitpushall.sh
 main_fragment.html
 toc_fragment.html
 
+# man pages
+docs/*.1
```
Comment (jubalh) [inline .gitignore:68]: why? :) I like to see those in `git status`

**Review by jubalh — CHANGES_REQUESTED** [review]

File: Makefile.am (hunk 1 above)
Comment (sjaeckel) [reply Makefile.am]: better now? :joy:

Comment (jubalh) [reply Makefile.am]: yes :)

File: .gitignore (hunk 2 above)
Comment (sjaeckel) [reply .gitignore:68]: because they are auto-generated!? :)

Comment (jubalh) [reply .gitignore:68]: Not all of them. Only the `profanity-*.1`.
And also I add them for each "release commit" and remove them later. And I like to see that with `git status` :)

Comment (sjaeckel) [reply .gitignore:68]: > Not all of them. Only the `profanity-*.1`.

OK, fair point, I'll update the ignore to `profanity-*.1`, OK?

or do I have to remove it completely?

Comment (jubalh) [reply .gitignore:68]: Since i add them in each release commit I would prefer to remove this line completey. Just so I see this and don't miss anything.

**Review by jubalh — APPROVED** [review]

**Comment by jubalh** [comment]
Coding style CI is really cool :)

**Pull request merged**